nu_stats/fqutils.py

```diff
@@ -0,0 +1,772 @@
+from collections import OrderedDict
```

Comment:
This file is getting long, maybe define the likelihoods elsewhere and import in to make it more manageable.

Reply:
On it!